server/services/prismic-parsers.js

```diff
@@ -18,6 +18,8 @@ import type {Series} from '../model/series';
 import type {LicenseType} from '../model/license';
 import {licenseTypeArray} from '../model/license';
 import {london} from '../filters/format-date';
+// $FlowFixMe
+import {parseContributors as parseContributorsProperly} from '../../common/services/prismic/parsers';
```

Comment:
Why the alias? Do we have another parseContributors?

Reply:
Yip - exactly that...
I'm trying to move out of this double land as much as possible with these PRs, but the chunk will be done when we move exhibitions and events into the whats_on service.